### PR TITLE
fix scala test compile flakes by forcing zinc scala version in select…

### PR DESCRIPTION
**Pull request by jack-berg**
The builds have been flakely lately with several modules' test scala compilation failing. After quite a bit of digging and trial and error, I've determined that the root cause is that for certain modules, the version of scala that the zinc incremental compiler uses somehow is resolved incorrectly. The scala version resolution failure for zinc is not deterministic, and seems isolated to just a couple modules.

If we see this type of error again, we should be able to fix it in a similar fashion by forcing the scala library version used for the `zinc` configuration.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -36,6 +36,7 @@ subprojects {
     repositories {
         mavenLocal()
         mavenCentral()
+        jcenter()
```
Comment (jack-berg) [inline build.gradle:39]: I routinely get resolution errors locally and adding jcenter() resolves them. I see that jcenter() is added as a repository elsewhere in the project, so I don't see any issue in adding another option for resolving dependencies.

File: instrumentation/akka-http-core-10.0/build.gradle [hunk 2]
```diff
@@ -9,7 +9,6 @@ dependencies {
     implementation("org.scala-lang:scala-library:2.11.12")
     implementation("com.typesafe.akka:akka-http-core_2.11:10.0.0")
     implementation("com.typesafe.akka:akka-actor_2.11:2.3.14")
-    testImplementation(project(":instrumentation:akka-2.2")) { transitive = false }
```
Comment (jack-berg) [inline instrumentation/akka-http-core-10.0/build.gradle:12]: Unused.

File: instrumentation/akka-http-core-2.11_10.0.11/build.gradle [hunk 3]
```diff
@@ -8,8 +8,6 @@ dependencies {
     implementation(project(":agent-bridge"))
     implementation("com.typesafe.akka:akka-http-core_2.11:10.0.11")
     implementation("com.typesafe.akka:akka-stream_2.11:2.5.11")
-
-    testImplementation(project(":instrumentation:akka-2.2")) { transitive = false }
```
Comment (jack-berg) [inline instrumentation/akka-http-core-2.11_10.0.11/build.gradle:12]: Unused.

File: instrumentation/akka-http-core-10.0.11/src/main/scala/akka/http/DefaultParsingErrorHandler.scala [hunk 4]
```diff
@@ -1,8 +0,0 @@
-package akka.http
-
-import com.newrelic.api.agent.weaver.SkipIfPresent
-
-@SkipIfPresent
-class DefaultParsingErrorHandler {
-
-}
```
Comment (jack-berg) [inline instrumentation/akka-http-core-10.0.11/src/main/scala/akka/http/DefaultParsingErrorHandler.scala:8]: This file was a remnant from a deleted module.

**Review by XiXiaPdx — APPROVED** [review]
Great find!!! Needle in a haystack.  

Any idea how it may have been introduced?

**Review by tspring — APPROVED** [review]

**Pull request merged**